tests/bundle/bundle-test.ts

```diff
@@ -118,114 +56,72 @@ async function runAll() {
 
   // Accounts
   console.log('Account setup started')
-  const faucet = {
-    publicKey: new Uint8Array([
-      238, 93, 102, 137, 215, 142, 38, 187, 91, 53, 176, 68, 23, 64, 160, 101,
-      199, 189, 142, 253, 209, 193, 84, 34, 7, 92, 63, 43, 32, 33, 181, 210,
-    ]),
-    secretKey: new Uint8Array([
-      205, 253, 96, 36, 210, 176, 235, 162, 125, 84, 204, 146, 164, 76, 217,
-      166, 39, 198, 155, 45, 189, 161, 94, 215, 229, 128, 133, 66, 81, 25, 174,
-      3,
-    ]),
-  }
+  const FaucetSeed =
+    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together'
+  const payerKp = generateKeypair({ seed: FaucetSeed, type: 'ed25519' })
   const payerSigner = await signerFromKeypair<'Ed25519', KiltAddress>({
-    keypair: faucet,
+    keypair: payerKp,
     algorithm: 'Ed25519',
   })
 
   console.log('faucet signer created')
 
-  const { didDocument: alice, getSigners: aliceSign } = await createFullDid(
+  const { identity: alice } = await createFullDidIdentity(
     payerSigner,
-    {
-      publicKey: new Uint8Array([
-        136, 220, 52, 23, 213, 5, 142, 196, 180, 80, 62, 12, 18, 234, 26, 10,
-        137, 190, 32, 15, 233, 137, 34, 66, 61, 67, 52, 1, 79, 166, 176, 238,
-      ]),
-      secretKey: new Uint8Array([
-        171, 248, 229, 189, 190, 48, 198, 86, 86, 192, 163, 203, 209, 129, 255,
-        138, 86, 41, 74, 105, 223, 237, 210, 121, 130, 170, 206, 74, 118, 144,
-        145, 21,
-      ]),
-    }
+    '//Alice',
+    'ed25519'
   )
   console.log('alice setup done')
 
-  const { didDocument: bob, getSigners: bobSign } = await createFullDid(
+  const { identity: bob } = await createFullDidIdentity(
     payerSigner,
-    {
-      publicKey: new Uint8Array([
-        209, 124, 45, 120, 35, 235, 242, 96, 253, 19, 143, 45, 126, 39, 209, 20,
-        192, 20, 93, 150, 139, 95, 245, 0, 97, 37, 242, 65, 79, 173, 174, 105,
-      ]),
-      secretKey: new Uint8Array([
-        59, 123, 96, 175, 42, 188, 213, 123, 164, 1, 171, 57, 143, 132, 244,
-        202, 84, 189, 107, 33, 64, 210, 80, 63, 188, 243, 40, 101, 53, 254, 63,
-        241,
-      ]),
-    }
+    '//Bob',
+    'ed25519'
   )
 
   console.log('bob setup done')
 
-  // Light DID Account creation workflow
-  const authPublicKey =
-    '0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
-
-  // const encPublicKey =
-  //   '0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb'
-
+  const authPublicKey = new Uint8Array([
+    170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170,
+    170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170, 170,
+    170, 170,
```

Comment:
Would this nice `.fill()` method be helpful here?

Reply:
ahh that's a good point!